### PR TITLE
created sign up ui

### PR DESCRIPTION
**Pull request by djpara**

### CODE REVIEW

**Comment by djpara** [comment]
Removed comments and TODO: markers. UI implemented with Bootstrap. Will need to refactor styling everywhere else in the app. Need to complete database save. Coordinating with Mark on that

File: imports/api/profiles.js [hunk 1]
```diff
@@ -1,23 +1,115 @@
...
+        allowedValues: ['{AL} Alabama', '{AK} Alaska', '{AZ} Arizona', '{AR} Arkansas', '{CA} California', '{CO} Colorado', '{CT} Connecticut', '{DE} Delaware', '{DC} District Of Columbia', '{FL} Florida', '{GA} Georgia', '{HI} Hawaii', '{ID} Idaho', '{IL} Illinois', '{IN} Indiana', '{IA} Iowa', '{KS} Kansas', '{KY} Kentucky', '{LA} Louisiana', '{ME} Maine', '{MD} Maryland', '{MA} Massachusetts', '{MI} Michigan', '{MN} Minnesota', '{MS} Mississippi', '{MO} Missouri', '{MT} Montana', '{NE} Nebraska', '{NV} Nevada', '{NH} New Hampshire', '{NJ} New Jersey', '{NM} New Mexico', '{NY} New York', '{NC} North Carolina', '{ND} North Dakota', '{OH} Ohio', '{OK} Oklahoma', '{OR} Oregon', '{PA} Pennsylvania', '{RI} Rhode Island', '{SC} South Carolina', '{SD} South Dakota', '{TN} Tennessee', '{TX} Texas', '{UT} Utah', '{VT} Vermont', '{VA} Virginia', '{WA} Washington', '{WV} West Virginia', '{WI} Wisconsin', '{WY} Wyoming']
+    },
+    postalCode: {
+        type: Number,
+        label: "Postal Code"
+    },
+    country: {
+        type: String,
+        label: "Country",
+        allowedValues: ['Not started', 'Travel', 'Entertainment', 'Sports']
```
Comment (alredi) [inline imports/api/profiles.js]: These are categories, not countries. If we're only allowing states right now, let's not worry about a country?

File: imports/api/profiles.js [hunk 2]
```diff
@@ -1,23 +1,115 @@
...
+    },
+    password: {
+        type: String,
+        label: "Password",
+        max: 100
+    },
+    gender: {
+        type: String,
+        label: "Gender",
+        allowedValues: ['Female', 'Male'],
```
Comment (alredi) [inline imports/api/profiles.js]: Let's add an option for 'Undisclosed'

File: imports/api/profiles.js [hunk 3]
```diff
@@ -1,23 +1,115 @@
...
+
+        update: function (userId, doc, fieldNames, modifier) {
+            return false;
+        },
+
+        remove: function (userId, doc) {
+            return false;
+        }
+    });
+
```
Comment (alredi) [inline imports/api/profiles.js]: Why are we allowing and denying the same functions? (lines 7-19 vs. lines 21-33)

**Review by alredi — CHANGES_REQUESTED** [review]

**Review by alredi — APPROVED** [review]

**Pull request merged**